universal-application-tool-0.0.1/conf/routes

```diff
@@ -24,6 +24,10 @@ POST    /admin/programs             controllers.admin.AdminProgramController.cre
 GET     /demo/new                   controllers.J2HtmlDemoController.newOne(request: Request)
 POST    /demo                       controllers.J2HtmlDemoController.create(request: Request)
 
+# Methods for applicants
+GET /applicants/:id/programs        controllers.applicant.ApplicantProgramsController.index(id: Long)
+GET /applicants/:aid/programs/:pid  controllers.applicant.ApplicantProgramsController.edit(aid: Long, pid: Long)
```

Comment:
nit: route for edit action should be `/applicants/:id/programs/:id/edit`

suggestion (non-blocking): recommend spelling the IDs out for readability e.g. `applicantId: Long, programId: Long`

Reply:
Done for both - thanks for the reminder on `edit`